docs/installation_configuration_guide/Configuration.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
I understand we will put content here later on right ?

Reply:
I added main part